src/main/java/school/hei/haapi/authentication/AwsCognitoJwtAuthFilter.java

```diff
@@ -0,0 +1,37 @@
+package school.hei.haapi.authentication;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+import org.springframework.stereotype.Component;
+
+import javax.servlet.*;
+import javax.servlet.http.HttpServletRequest;
+import java.io.IOException;
+@Component
+public class AwsCognitoJwtAuthFilter extends GenericFilter {
+
+    private static final Log logger = LogFactory.getLog(AwsCognitoJwtAuthFilter.class);
```

Comment:
1. Caps for constans
2. Use @slf4j of Lombok

Reply:
Please refer to my comment about the use of checkstyle instead of Google Java Style in Sonar.

Reply:
constant = static final

I see that you put in caps lock final variables. Please revert them to caps unlock. Only variables that are both static and final are called constants.

Reply:
Ah ok.
I was confused about the use of immutable value ...